indra/newview/llpanelprofile.cpp

```diff
@@ -719,7 +719,7 @@ BOOL LLPanelProfileSecondLife::postBuild()
     mGroupList              = getChild<LLGroupList>("group_list");
     mShowInSearchCombo      = getChild<LLComboBox>("show_in_search");
     mHideAgeCombo           = getChild<LLComboBox>("hide_age");
-    mSecondLifePic          = getChild<LLThumbnailCtrl>("2nd_life_pic");
+    mSecondLifePic          = getChild<LLIconCtrl>("2nd_life_pic");
```

Comment:
Ideally LLIconCtrl should be moved into LLProfileImageMonitor, but I'm not sure how to amke that gracefully, may be I should just make a whole new Ctrl...